LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -120,7 +120,7 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
             content
             Button(action: { self.onSave?(self.viewModel.insulinModelSettings) }) {
                 Text(mode.buttonText)
-                    .buttonStyle(ActionButtonStyle(.primary))
+                    .actionButtonStyle(.primary)
```

Comment:
@mpangburn this actually broke the button appearance, so I'm reverting it.  `.actionButtonStyle` is still used in a lot of other places...if it is deprecated I'd like to wait to replace it if you don't mind. 
(and, no, moving the `.buttonStyle(ActionButtonStyle(.primary))` to apply to the `Button` still didn't look right, it will require more tweaking.

Reply:
Huh - what didn't look right? I know I have a really similar snippet of code in my `InsulinModelReview`

Reply:
`InsulinModelReview` uses `.actionButtonStyle(.primary)` on the `Text` object too, and it looks fine.  @mpangburn had suggested in my previous PR that I change it to `.buttonStyle(ActionButtonStyle(.primary))` but I put it in the wrong place here. After moving it to the `Button` object, it now looked too big (and inconsistent with all the other screens).  Therefore, if we're deprecating `.actionButtonStyle(.primary)` then I would prefer to make a blanket change to all the screens so the look is consistent, not just do it here.  Make sense?

Reply:
Ahh, gotcha.